api/docs/release.dox

```diff
@@ -241,6 +241,9 @@ Further non-compatibility-affecting changes include:
    trace analysis tool framework.
```

Comment:
> The filter is enabled by -filter_func_markers_by_value followed by a list of integers that represent the function IDs of TRACE_MARKER_TYPE_FUNC_ markers to keep in the trace.

To keep?  That's the opposite of the other -filter options where you specify what to *remove*.

Reply:
Right, but for our use case we have a lot more function IDs to remove (much more than TRACE_MARKER_TYPE_ enum values, which type_filter operates on), so it's more convenient to list the ones we want to keep.
We could add a second option to turn on func_id_filter_t: `-filter_func_ids` (as opposed to the current `-filter_keep_func_ids`) which would remove the markers related to the listed function IDs (these two options would be mutually exclusive).
But perhaps it makes sense to do so when we have a need for it?